Detectors/GlobalTrackingWorkflow/tofworkflow/src/TOFMatchedReaderSpec.cxx

```diff
@@ -0,0 +1,105 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file   TOFMatchedReaderSpec.cxx
+
+#include <vector>
+
+#include "TTree.h"
+#include "TFile.h"
+
+#include "TOFWorkflow/TOFMatchedReaderSpec.h"
+#include "Framework/ControlService.h"
+#include "Framework/ConfigParamRegistry.h"
+#include "Headers/DataHeader.h"
+#include "SimulationDataFormat/MCCompLabel.h"
+#include "SimulationDataFormat/MCTruthContainer.h"
+#include "ReconstructionDataFormats/MatchInfoTOF.h"
+
+using namespace o2::framework;
+
+namespace o2
+{
+namespace tof
+{
+
+void TOFMatchedReader::init(InitContext& ic)
```

Comment:
Could you please add to this reader changes similar to those in the [PR](https://github.com/AliceO2Group/AliceO2/pull/3334) I've just opened, to be able to process multiple TF entries. They all now behave in a same way, take e.g. [TPC/ITS matches reader](https://github.com/AliceO2Group/AliceO2/pull/3334/files#diff-bc630e5b1e5bd9c876a97b4084a8541c) modifs as an example.